lib/gui/models/store.js

```diff
@@ -336,21 +336,6 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
         });
       }
 
-      if (!action.data.extension) {
-        throw errors.createError({
-          title: 'Missing image extension'
```

Comment:
If this is no longer an error, I guess we should add some extension-less test files to the unit-tests?